assignment3/functions.dart

```diff
@@ -0,0 +1,132 @@
+import 'dart:io';
+import 'graph.dart';
+import 'util.dart';
+
+void printOptions() {
```

Comment:
Don't add functions like these directly. Create a class and add these as static methods to it.

Reply:
created a Function class and added as static functions